specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-06-01/containerregistry_build.json

```diff
@@ -0,0 +1,2577 @@
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/Run"
+            }
+          },
+          "202": {
+            "description": "The request was successfully accepted; the operation will complete asynchronously."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If the registry/run doesn't exist, 404 (Not found) is returned. If any of the input parameters is wrong, 400(Bad Request) is returned."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Registries_ScheduleRun": {
+            "$ref": "./examples/RegistriesScheduleRun.json"
+          },
+          "Registries_ScheduleRun_EncodedTaskRun": {
+            "$ref": "./examples/RegistriesScheduleRun_EncodedTaskRun.json"
+          },
+          "Registries_ScheduleRun_FileTaskRun": {
+            "$ref": "./examples/RegistriesScheduleRun_FileTaskRun.json"
+          },
+          "Registries_ScheduleRun_Task": {
+            "$ref": "./examples/RegistriesScheduleRun_Task.json"
+          },
+          "Registries_ScheduleRun_Task_WithCustomCredentials": {
+            "$ref": "./examples/RegistriesScheduleRun_FileTask_WithCustomCredentials.json"
+          },
+          "Registries_ScheduleRun_WithCustomCredentials": {
+            "$ref": "./examples/RegistriesScheduleRun_WithCustomCredentials.json"
+          }
+        }
+        }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}/listBuildSourceUploadUrl": {
+      "post": {
+        "tags": [
+          "Registries"
+        ],
+        "description": "Get the upload location for the user to be able to upload the source.",
+        "operationId": "Registries_GetBuildSourceUploadUrl",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/SourceUploadDefinition"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If the registry doesn't exist, 404 (Not found) is returned."
+          }
+        },
+        "x-ms-examples": {
+          "Registries_GetBuildSourceUploadUrl": {
+            "$ref": "./examples/RegistriesGetBuildSourceUploadUrl.json"
+          }
+        }
```

Comment:
Compared to the previous version, there are lot of apis that got dropped.. Could you add details on why those got dropped to the description. ?
for ex: collection get call on following URi is not present.. & /CheckNameAvailability is dropped.
GET /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}

[https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL2dpdGh1Yi5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvYmxvYi9tYXN0ZXIvc3BlY2lmaWNhdGlvbi9jb250YWluZXJyZWdpc3RyeS9yZXNvdXJjZS1tYW5hZ2VyL01pY3Jvc29mdC5Db250YWluZXJSZWdpc3RyeS9zdGFibGUvMjAxOS0wNS0wMS9jb250YWluZXJyZWdpc3RyeS5qc29uIiwidGFyZ2V0VXJsIjoiaHR0cHM6Ly9naXRodWIuY29tL0F6dXJlL2F6dXJlLXJlc3QtYXBpLXNwZWNzL2Jsb2IvYTlhYjU1NzNhMmY1OTk2MzVlNjMwYzYwOWEzYmU4NzY0MDFhZGFmZi9zcGVjaWZpY2F0aW9uL2NvbnRhaW5lcnJlZ2lzdHJ5L3Jlc291cmNlLW1hbmFnZXIvTWljcm9zb2Z0LkNvbnRhaW5lclJlZ2lzdHJ5L3N0YWJsZS8yMDE5LTA2LTAxL2NvbnRhaW5lcnJlZ2lzdHJ5X2J1aWxkLmpzb24ifQ==](https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL2dpdGh1Yi5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvYmxvYi9tYXN0ZXIvc3BlY2lmaWNhdGlvbi9jb250YWluZXJyZWdpc3RyeS9yZXNvdXJjZS1tYW5hZ2VyL01pY3Jvc29mdC5Db250YWluZXJSZWdpc3RyeS9zdGFibGUvMjAxOS0wNS0wMS9jb250YWluZXJyZWdpc3RyeS5qc29uIiwidGFyZ2V0VXJsIjoiaHR0cHM6Ly9naXRodWIuY29tL0F6dXJlL2F6dXJlLXJlc3QtYXBpLXNwZWNzL2Jsb2IvYTlhYjU1NzNhMmY1OTk2MzVlNjMwYzYwOWEzYmU4NzY0MDFhZGFmZi9zcGVjaWZpY2F0aW9uL2NvbnRhaW5lcnJlZ2lzdHJ5L3Jlc291cmNlLW1hbmFnZXIvTWljcm9zb2Z0LkNvbnRhaW5lclJlZ2lzdHJ5L3N0YWJsZS8yMDE5LTA2LTAxL2NvbnRhaW5lcnJlZ2lzdHJ5X2J1aWxkLmpzb24ifQ==)

Reply:
Hi @sanjaiganesh , the previous API version is `2019-04-01` for ACR Tasks. 
Some of the API versions belong to ACR RP (for instance, 2019-05-01 that you compared), and some of them belong to ACR Task RP (previous version 2019-04-01). 
cc: @jaysterp in case she can better explain this.